python/paddle/fluid/contrib/slim/quantization/quantization_pass.py

```diff
@@ -917,9 +924,8 @@ def __init__(self,
         self._scope = scope
         self._place = place
         self._quantizable_ops = quantizable_op_type
-        supported_quantizable_ops = ['conv2d', 'depthwise_conv2d', 'mul']
         for op in self._quantizable_ops:
-            assert op in supported_quantizable_ops, \
+            assert op in QuantizationTransformPass.supported_quantizable_op_type, \
```

Comment:
`assert op in ConvertToInt8Pass._supported_quantizable_op_type, \`

Reply:
> `assert op in ConvertToInt8Pass._supported_quantizable_op_type, \`

Done